garak/detectors/misleading.py

```diff
@@ -66,13 +66,16 @@ class MustContradictNLI(HFDetector):
     def detect(self, attempt: Attempt) -> List[float]:
         import torch
 
-        premise = attempt.prompt_for(self.lang_spec)
+        premise = attempt.prompt_for(self.lang_spec).last_message().text
         all_outputs = attempt.outputs_for(self.lang_spec)
         results = []
         for hypothesis in all_outputs:
+            if not hypothesis:
+                results.append(0.0)
```

Comment:
This results in a behavior change in error cases, where `None` is returned from the generator.

```
misleading.FalseAssertion                                               misleading.MustContradictNLI: SKIP  ok on    0/   0
```
Becomes
```
misleading.FalseAssertion                                               misleading.MustContradictNLI: PASS  ok on  150/ 150
```

Reply:
```suggestion
```

Confirmed to maintain current behavior not result should be appended for `None` outputs.